net/CMakeLists.txt

```diff
@@ -1,2 +1 @@
-add_subdirectory(golioth)
-add_subdirectory(wifi)
+add_subdirectory(golioth)
```

Comment:
There is "No newline at end of file" warning shown by GitHub. The common convention is to always add newlines at the end of text files. I've created #135, which will enforce that in CI.